src/content/docs/accounts/accounts/automated-user-management/tutorial-manage-users-groups-scim.mdx

```diff
@@ -0,0 +1,694 @@
+* `userName` This identifier **must** be unique within your Authentication Domain. Use the user's email address. 
+* `emails` Email address of the user.
+* `active` Boolean indicating whether or not the user should be active or inactive within New Relic.
+
+It is recomended to provide the following attributes for the best user experience:
+* `name.givenName` The user's first or given name.
+* `name.familyName` The user's last or family name.
+* `timezone` The user's timezone in IANA Time Zone database format.
+
+Take note of the returned user `id`. To update a user in the future you will need to supply the same ID with the request.
+
+```
+curl -X 'POST' -H 'Content-Type: application/json' -H "Authorization: Bearer $TOKEN" 'https://scim-provisioning.service.newrelic.com/scim/v2/Users' --data-binary @- <<EOF
+{
+  "schemas": ["urn:ietf:params:scim:schemas:core:2.0:User"],
+  "userName": "bjensen@example.com",
+  "name": {
+    "familyName": "Jensen",
+    "givenName": "Barbara"
+  },
+  "emails": [
+    {
+      "primary": true,
+      "value": "bjensen@example.com"
+    }
+  ],
+  "active": true,
+  "timezone": "America/Los_Angeles"
+}
+EOF
+```
+
+Example responses:
+
+<CollapserGroup>
+  <Collapser title="201 Created">
+  ```
+  {
+    "schemas": [
+      "urn:ietf:params:scim:schemas:core:2.0:User",
+    ],
+    "id": "5a1d580f-323c-450c-8c62-479b5c9085d6",
+    "externalId": null,
+    "userName": "bjensen@example.com",
+    "name": {
+      "familyName": "Jensen",
+      "givenName": "Barbara"
+    },
+    "emails": [
+      {
+        "value": "bjensen@example.com",
+        "primary": true
+      }
+    ],
+    "timezone": "America/Los_Angeles",
+    "active": true,
+    "meta": {
+      "resourceType": "User",
+      "created": "2021-07-20T21:32:58.167Z",
+      "lastModified": "2021-07-20T21:32:58.167Z"
+    },
+    "groups": []
+  }
+  ```
+  </Collapser>
+  
+  <Collapser title="400 Bad Request">
+  ```
+  {
+    "schemas": [
+      "urn:ietf:params:scim:api:messages:2.0:Error"
+    ],
+    "scimType": "invalidValue",
+    "detail": "Validation failed: Username can't be blank",
+    "status": "400"
+  }
+  ```
+  </Collapser>
+  
+  <Collapser title="409 Conflict">
+  ```
+  {
+    "schemas": [
+      "urn:ietf:params:scim:api:messages:2.0:Error"
+    ],
+    "scimType": "uniqueness",
+    "detail": "Resource already exists",
+    "status": "409"
+  }
+  ```
+  </Collapser>
+</CollapserGroup>
+
+## Create groups in your Authentication Domain [#create-groups]
+
+If you are using a third-party identity provider, refer to their documentation for how to push groups with SCIM.
+
+Otherwise, send a `POST` request to `/scim/v2/Groups` to create a group. The only group attribute **required** is:
+* `displayName` The group name.
+
+Take note of the returned group `id`. To update a group or its members in the future you will need to supply the same ID with the request.
```

Comment:
Again, probably needs to be a `<Callout>` or maybe separate this from the examples below with a header.